lsql/judge/tests/test_views.py

```diff
@@ -290,6 +327,152 @@ def test_download(self):
+        self.assertTrue(response.status_code == 404 and msg in str(response.content))
+        client.logout()
+        client.login(username=user_1.username, password='12345')
+
+        # me intento conectar con el usuario pepe a 1B que no puedo
+        response = client.get(classification_url, {'group': group_b.id}, follow=True)
+        self.assertTrue(response.status_code == 403)
+
+        # Hago un fallo del primer ejercicio y al segundo intento acierto
+        client.post(submit_select_url, {'code': 'SELECT * FROM test where n = 1000'}, follow=True)
+        client.post(submit_select_url, {'code': select_problem.solution}, follow=True)
+
+        # Hago 3 fallos del segundo ejercicio y al cuarto hacierto
+        client.post(submit_dml_url, {'code': 'SELECT * FROM test where n = 1000'}, follow=True)
+        client.post(submit_dml_url, {'code': 'SELECT * FROM test where n = 1000'}, follow=True)
+        client.post(submit_dml_url, {'code': 'SELECT * FROM test where n = 1000'}, follow=True)
+        client.post(submit_dml_url, {'code': dml_problem.solution}, follow=True)
+
+        # me meto a 1A y compruebo que encuentro 1/2 (2) y 1/4 (4)
+        # Puntuacion = 6 y Resueltos 2
+        response = client.get(classification_url, {'group': group_a.id}, follow=True)
+        self.assertTrue(response.status_code == 200 and '1/2 (2)' in str(response.content))
+        self.assertTrue(response.status_code == 200 and '1/4 (4)' in str(response.content))
+        self.assertTrue(response.status_code == 200 and '6' in str(response.content))
+        self.assertTrue(response.status_code == 200 and '2' in str(response.content))
+
+        client.logout()
+        client.login(username=user_2.username, password='12345')
+
+        # Hago del primer ejercicio 3 aciertos de 3 intentos (3/3 (1))
+        client.post(submit_select_url, {'code': select_problem.solution}, follow=True)
+        client.post(submit_select_url, {'code': select_problem.solution}, follow=True)
+        client.post(submit_select_url, {'code': select_problem.solution}, follow=True)
+
+        # Hago del segundo ejercicio, dos fallos y despues acierto (1/3 (3))
+        client.post(submit_dml_url, {'code': 'Select * from test'}, follow=True)
+        client.post(submit_dml_url, {'code': 'Select * from test'}, follow=True)
+        client.post(submit_dml_url, {'code': dml_problem.solution}, follow=True)
+
+        response = client.get(classification_url, {'group': group_a.id}, follow=True)
+        self.assertTrue(response.status_code == 200 and '3/3 (1)' in str(response.content))
+        self.assertTrue(response.status_code == 200 and '1/3 (3)' in str(response.content))
+        self.assertTrue(response.status_code == 200 and '4' in str(response.content))
+        self.assertTrue(response.status_code == 200 and '2' in str(response.content))
+
+        # se comprueba que ana va primera y pepe segundo aunque tengan mismos resueltos
```

Comment:
Este comentario parece que introduce una comprobación que no se llega a realizar

Reply:
Mirar si se puede relajar la longitud máxima en Pylint

Reply:
¿Cómo es que pylint te ha dejado añadir más de 70 líneas? ¿Has hecho algo especial?

Reply:
Me refería a 70 self.assert, como máximo deja 70 assertTrue, Equals, In, etc, líneas si me deja más, lo que me pasaba que tenía 75 assert y tuve que poner 70

Reply:
Ok, pues mejor. Pero ya sabemos para el futuro que no hay que quitar `asserts`, si se queja Pylint se divide el test de unidad grande en dos y listo.